### DISCUSSION TITLE
How i can always show placeholder?

### DISCUSSION THREAD

**Discussion by dinfyru** [JedWatson/react-select#5036] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/10163112/152290477-11c6927b-2cd4-4e9e-8a74-df030a7ae51d.png)
i want do like this effect. Placeholder goes up when value is selected or input value is not empty

**Comment by Rall3n** [comment, marked as answer]
@dinfyru a quick search through existing issues would have given you #4221 and #2152.

I'm not gonna summarize the issues as the solutions in both are very well explained.
